changelog/12765.feature.rst

```diff
@@ -0,0 +1 @@
+Thresholds to trigger snippet truncation can now be set with `--truncation-limit-lines` and `--truncation-limit-chars` command line options.
```

Comment:
Nice 👍 

Perhaps we should also update [`output.rst`](doc/en/how-to/output.rst) to mention the new command-line options (or ini options as per my comment).